src/components/PopupWithForm.js

```diff
@@ -22,6 +22,11 @@ export default class PopupWithForm extends Popup {
     return inputInfo;
   }
 
+  setInputValues({name, about}) {
+    this.#form.querySelector("#profile-input-username").value = name;
+    this.#form.querySelector("#profile-input-about").value = about;
+  }
```

Comment:
Please, delete this hardcode from universal `PopupWithForm`


It's only for the proifle modal 

There should **not** be any code in universal functions that doesn’t suit every popup.